src/js/chamber.coffee

```diff
@@ -46,3 +50,20 @@ class Chamber extends THREE.Mesh
 
   calculateGeometryRing: ->
     vertex for vertex in @.geometry.vertices when vertex.z == 0
+
+  buildThicknessVector: ->
+    direction = new THREE.Vector3 0, 1, 0
+
+    thicknessVector = new THREE.ArrowHelper direction, @origin, @thickness, @GROWTH_VECTOR_COLOR
+    thicknessVector.visible = false
+
+    thicknessVector
+
+  showThicknessVector: ->
+    @thicknessVector.visible = true
+
+  hideThicknessVector: ->
+    @thicknessVector.visible = false
+
+  toggleThicknessVector: ->
+    @thicknessVector.visible != @thicknessVector.visible
```

Comment:
To. Ale czy to jest w ogóle używane? Wydaje mi się, że używasz tylko Simulation.toggleThicknessVector**s**, które jest napisane poprawnie